guides/common/modules/proc_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -1,31 +1,32 @@
 [id="configuring-compliance-policy-deployment-methods_{context}"]
 = Configuring Compliance Policy Deployment Methods
 
-Use the following procedure to configure {Project} for the method that you have selected to deploy compliance policies.
-
-.Procedure
-* For the Ansible deployment method:
-** Import the `theforeman.foreman_scap_client` Ansible role.
+Use one the following procedures to configure {Project} for the method that you have selected to deploy compliance policies.
 
+.Procedure for Ansible deployment
+. Import the `theforeman.foreman_scap_client` Ansible role.
 +
 For more information, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
-
+. Assign the created policy and the `theforeman.foreman_scap_client` Ansible role to a host or host group.
+. To trigger the deployment, run the Ansible role on the host or host group either manually, or set up a recurring job using remote execution for regular policy updates.
 +
-Then assign the created policy and the `theforeman.foreman_scap_client` Ansible role to a host or host group.
-To trigger the deployment, run the Ansible role on the host or host group either manually, or set up a recurring job using remote execution for regular policy updates.
 For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
-* For the Puppet deployment method:
+
+.Procedure for Puppet deployment
 . Ensure Puppet is enabled.
 . Ensure the Puppet agent is installed on hosts.
 . Import the Puppet environment that contains the `foreman_scap_client` Puppet module.
-For more information, see {ManagingConfigurationsPuppetDocURL}[_{ManagingConfigurationsPuppetDocTitle}_].
-
 +
-Then assign the created policy and the `foreman_scap_client*` Puppet classes to a host or host group.
+For more information, see {ManagingConfigurationsPuppetDocURL}[_{ManagingConfigurationsPuppetDocTitle}_].
+. Assign the created policy and the `foreman_scap_client*` Puppet classes to a host or host group.
 Puppet triggers the deployment on the next regular run or you can run Puppet manually.
-Note that Puppet runs every 30 minutes by default.
++
+NOTE: Puppet runs every 30 minutes by default.
```

Comment:
I am unsure about adding an admonition here. Any "+1" or "-1" from someone else?

Reply:
+1
We generally try to avoid them so that the docs don't become "over-admonitioned" :D

Reply:
"Note that ..." in the first version is an admonition in itself. So I don't see this as adding a new admonition; to me, this is just adding markup that had been missing.

We are in agreement of not overusing admonitions, so I'll rewrite this part to avoid it.